javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+const $heading = document.querySelector('h1');
+let count = 4;
+
+function countdown() {
+  if (count > 1) {
+    count--;
+    $heading.textContent = `${count}`;
```

Comment:
Works, but you can also just use `count` directly since you aren't concatenating it with any other string. `$heading.textContent = count`